setup/pi/create-backingfiles.sh

```diff
@@ -216,6 +221,15 @@ then
 fi
 
 REMAINING_SPACE="$(available_space)"
+
+if [ "$REMAINING_SPACE" -ge 1024 ] && [ "$LIGHTSHOW_DISK_SIZE" -gt 0 ]
```

Comment:
This check seems wrong. It's checking that there is more than 1MB available, but it seems likely that LIGHTSHOW_DISK_SIZE will be greater than 1MB